files/build_templates/sonic_debian_extension.j2

```diff
@@ -272,6 +272,19 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-device-data_*.deb || \
 # package for supporting password hardening
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libpam-pwquality
 
+# Install pam-ldap, nss-ldap, ldap-utils
```

Comment:
>Install [](http://example.com/codeflow?start=2&length=7)

Suggest to add build time config, so deployers not using LDAP could reduce image size, and reduce the security concerns.

Reply:
Why should this be different than radius/tacacs? Why show different approaches for a feature in the same domain?